### ISSUE TITLE
Improve Apollo repo pulling on install/remove Jobs 

### ISSUE THREAD

**Issue by P-147** [apollo-business-automation/ibm-cp4ba-enterprise-deployment#28]
## > Is your feature request related to a problem? Please describe

**TL;DR:** This feature will improve start speed (and footprint) of install & remove Jobs  

==> I noticed your repo is becoming quite big (lots of commits, some big files too... 😯  ).  
On the `README.md`, all YAMLS to launch the install/remove Jobs contain this line :

```bash
git clone --progress --branch ${GIT_BRANCH} ${GIT_REPOSITORY}
```  

==> This command clones ALL the **commit history** inside the container...

## > Describe the solution you'd like

As the commit history isn’t needed for running your scripts (we only want the last version of the scripts), you can add the 2 args `--depth 1 --shallow-submodules` to the git clone command to do a shallow clone ([cf. git documentation](https://git-scm.com/docs/git-clone/en#Documentation/git-clone.txt---depthltdepthgt)).  

So the command would become:

```bash
git clone --depth 1 --shallow-submodules --progress --branch ${GIT_BRANCH} ${GIT_REPOSITORY}
```
Results in image: 
<img width="1423" alt="Apollo-git-clone-improve" src="https://user-images.githubusercontent.com/43950165/218109962-1f59f865-b758-4daf-91b4-56e23fecf48b.png">
For the pull timing, we can easily gain 15-30sec between the 2 methods 🙂 

## > Additional context

*Modification Impacts*:  
I’ve noticed 3 places in your repo where a `git clone` command is used :

- `README.md` --> lines 599 & 738
- `scripts/servicing-pod.yaml` --> line 14
